src/arcade/patch/agent/cell/PatchCellCancer.java

```diff
@@ -29,6 +29,7 @@ public class PatchCellCancer extends PatchCellTissue {
      */
     public PatchCellCancer(PatchCellContainer container, Location location, Parameters parameters) {
         this(container, location, parameters, null);
+        MAX_DENSITY = Integer.MAX_VALUE;
```

Comment:
I think previously the max for cancer cells was 6, is there a reason why it is now the max int?

Reply:
It was an arbitrary limit based on the amount of subcoordinates in the location. Functionally the cells do not reach this number.

Reply:
I could definitely set it to 6 though but this seems a little more clear what the purpose is.

Reply:
no I agree that having it be max int makes it more clear